### PR TITLE
Add signing pipeline yaml to v5 branch

### LINKED ISSUE #6032: Create v5 release pipeline to Azure artifacts
## Summary

Once we have a pipeline for v5 in #6031, we also need a pipeline to publish v5 signed nightly builds to Azure artifacts.

Note: we should wait until #6026 is done before we start publishing so packages are properly versioned.

### PR DESCRIPTION
**Pull request by BruceHaley**
Fixes #6032

### CODE REVIEW

**Review by gabog — DISMISSED** [review]
Hi @BruceHaley, shouldn't we wait until #6026 is done before we merge this one?

File: build/yaml/botbuilder-dotnet-sign.yml
Comment (gabog) [inline build/yaml/botbuilder-dotnet-sign.yml:23]: Do we need to uncomment this lines to apply the right preview version? or that's done somewhere else?

**Review by gabog — COMMENTED** [review]
Monica already took care of updating the project versions. 

I only have a question, the rest looks good. 

Thanks

File: build/yaml/botbuilder-dotnet-sign.yml
Comment (mrivera-ms) [reply build/yaml/botbuilder-dotnet-sign.yml:23]: This is done somewhere else. We should remove these lines, however.

File: build/yaml/botbuilder-dotnet-sign.yml
Comment (mrivera-ms) [inline build/yaml/botbuilder-dotnet-sign.yml:2]: > Replaces the classic BotBuilder-DotNet-master-Signed-daily. [](http://example.com/codeflow?start=1&length=60)

This can be removed.

File: build/yaml/botbuilder-dotnet-sign.yml
Comment (mrivera-ms) [inline build/yaml/botbuilder-dotnet-sign.yml:24]: Please remove this.

File: build/yaml/botbuilder-dotnet-sign.yml
Comment (mrivera-ms) [inline build/yaml/botbuilder-dotnet-sign.yml:55]: This can be removed.

File: build/yaml/botbuilder-dotnet-sign.yml
Comment (BruceHaley) [reply build/yaml/botbuilder-dotnet-sign.yml]: I have reworded these comments for better clarity. (No, we don't uncomment them.)
These comments are important so pipeline maintainers will know what variables must be defined externally, and how they are consumed outside the pipeline. These comments are a standard element in all of our pipelines. Viz: https://github.com/microsoft/BotBuilder-Samples/blob/1417f920913824f61f694d25e18a325e07543e3a/build/yaml/sample-dotnet-samplebot-linux-test.yml#L6

File: build/yaml/botbuilder-dotnet-sign.yml
Comment (BruceHaley) [reply build/yaml/botbuilder-dotnet-sign.yml]: Reworded.

File: build/yaml/botbuilder-dotnet-sign.yml
Comment (BruceHaley) [reply build/yaml/botbuilder-dotnet-sign.yml]: Explained above.

File: build/yaml/botbuilder-dotnet-sign.yml
Comment (BruceHaley) [reply build/yaml/botbuilder-dotnet-sign.yml]: Done.

**Review by mrivera-ms — APPROVED** [review]

**Comment by BruceHaley** [comment]
> Hi @BruceHaley, shouldn't we wait until #6026 is done before we merge this one?

It's done.

**Pull request merged**
